docs/security/access-control.md

```diff
@@ -0,0 +1,160 @@
+|Project Administrators|Has permissions to administer all aspects of teams and project, although they can't create team projects.|Assign to users who manage user permissions, create or edit teams, modify team settings, define area an iteration path, or customize work item tracking.|**Platform Admin**|
+|{team name}|Has permissions to contribute fully to the project code base and work item tracking. The default Team group is created when you create a project, and by default is added to the Contributors group for the project. Any new teams you create will also have a group created for them and added to the Contributors group.|Add members of the team to this group.|**Data Scientist**|
+
+## How to restrict access to ADO for QA team?
+Simply Platform Admin should not give organization-level and project-level ADO permissions to QA team members.
+
+
+<a id="anchor1"></a>
+## <a href="#anchor1">Access Control - Azure ML</a>
+### Groups using Azure Active Directory
+Create two groups using Azure Active Directory.
+|Group Name|Attribute|
+|-|-|
+|{team name} for Dev members|Main Engineering group who needs to tune data and use Azure Machine Learning|
+|{team name} for QA members|Q&A teams who should access only test data|
+
+Each group has the following access:
+|Resource Name|{team name} for Dev members|{team name} for QA members|
+|-|-|-|
+|Subscription|Contributor|**No Access**|
+|Resource Group for dev|Contributor|**No Access**|
+|Resource Group for stage|Contributor|**No Access**|
+|[Function App "func-trigger-{BaseName}"](../../common/pipeline_trigger/README.md)|Contributor|**No Access**|
+|Storage Account for dev|Contributor|**No Access**|
+|Storage Account for stage|Contributor|**Reader**|
+
+## What QA team should NOT do?
+- Not see the details on Azure DevOps and Azure ML
+
+## What QA team should do?
+- Run a performance experiment pipeline
+- Add/Delete test data to/from folders "test-pcm", "test-meta"
+- See experiment results on Azure Dashboard
+- Receive notifications when experiment is started and ended via email/Microsoft Teams/Slack.
+## How to restrict access to AML for QA team?
+-	Access only specific folders for test data in storage
+-	Access Azure Dashboard to see performance experiment results
+-	Use managed service identity and Azure Function ([HTTP trigger](../../common/pipeline_trigger/README.md)) to run a performance experiment pipeline with key and test config file path
+
+*QA team cannot access to even Azure Function resource.
+
+*Azure Function can access to AML with managed service identity.
+
+
+## How to create custom role to restrict specific actions?
+Once you add an AAD group as contributor, people in the group can do almost anything for the resource.
+
+Creating custom role is another option to achieve the requirement in order to reduce permissions for QA team as much as possible.
+
+Please refer to [the wiki - custom role](/Spikes/Security/Custom-Role) to know more details.
```

Comment:
Is this link valid?

Reply:
Thank you, I just changed it to the appropriate MS docs.